salt/modules/swarm.py

```diff
@@ -0,0 +1,360 @@
+# -*- coding: utf-8 -*-
+"""
+Docker Swarm Module using Docker's Python SDK
+=============================================
+
+:codeauthor: Tyler Jones <jonestyler806@gmail.com>
+
+.. versionadded:: 2017.7.2
```

Comment:
This should be Oxygen, which is our next major release.

Reply:
Ok, easy fix.  :)